job.py

```diff
@@ -10,13 +11,15 @@
 def main():
     ns = os.getenv('EXPERIMENT_NAMESPACE')
     job_name = os.getenv('JOB_NAME')
+    log = logging.getLogger('experiments_job')
+
     if not ns or not job_name:
         raise Exception('Missing EXPERIMENT_NAMESPACE or JOB_NAME')
 
-    c = Client(ns)
+    c = Client(log, ns)
```

Comment:
I'm a bit puzzled why we're passing the same logger instance into the client constructor here instead of defining separate logger per module (as is common practice).

See for example: https://docs.python.org/2/howto/logging.html#advanced-logging-tutorial and the `logger = logging.getLogger(__name__)` statement.